classes/helper.php

```diff
@@ -57,6 +57,8 @@ public static function script_type_display(int $type): string {
                 return get_string('scripttype_ajax', 'tool_excimer');
             case profile::SCRIPTTYPE_WS:
                 return get_string('scripttype_ws', 'tool_excimer');
+            case profile::SCRIPTTYPE_TASK:
```

Comment:
I get this error on the profile.php detail page:

( ! ) Notice: Undefined offset: 0 in /var/www/moodle.local/admin/tool/excimer/classes/helper.php on line 125
--


1 | 0.0003 | 361904 | {main}( ) | .../profile.php:0
2 | 0.7364 | 17506720 | tool_excimer\helper::http_status_display( ) | .../profile.php:119

Reply:
fixed